bedevere/bpo.py

```diff
@@ -37,6 +40,19 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            if CLOSING_TAG not in body:
+                issue_number = issue_number_found.group("issue")
+                BPO_MSG = textwrap.dedent(f"""\
```

Comment:
Nope, I see what happened. Berker's body had newlines in it that led to parts of the body not being equally indented like the string literal, so when `textwrap.dedent()` got it there was whole chunks of text not indented so it assumed there was nothing to dedent. IOW it worked as intended and it just  didn't occur to me when doing the code review that the inserted body would have newlines followed by text that wouldn't be indented.